charts/civil-camunda/values.preview.template.yaml

```diff
@@ -18,7 +18,6 @@ java:
         - cross-access-user-password
         - prd-admin-user-username
         - prd-admin-user-password
-        - AppInsightsInstrumentationKey
```

Comment:
I thought the key you need for logging was appinsights-connection-string not this